data/json/scenarios.json

```diff
@@ -867,6 +877,139 @@
     "start_name": "Nuclear Power Plant",
     "allowed_locs": [ "nuclear_plant" ],
     "professions": [ "electrician", "labtech", "medic" ],
+    "requirement": "achievement_reach_nuclear_plant",
     "flags": [ "CHALLENGE", "LONE_START" ]
+  },
+  {
+    "type": "scenario",
+    "name": "Challenge - Close Encounter",
+    "description": "You don't know what it was but you managed to lose it for the time being.  Now you are injured, surrounded and the building is on fire.  You hope you have some time before it returns to finish the job.",
+    "flags": [ "CITY_START", "CHALLENGE", "LONE_START", "HELI_CRASH" ],
+    "id": "close_encounter_start",
+    "points": -5,
+    "start_name": "In Town",
+    "forced_traits": [ "HAS_NEMESIS" ],
+    "eoc": [ "scenario_surrounded_zombie", "EOC_scenario_fire", "EOC_scenario_smoke" ],
+    "requirement": "achievement_kill_100_monsters",
+    "allowed_locs": [
+      "sloc_grocery_store",
+      "sloc_garage",
+      "sloc_furniture_store",
+      "sloc_library",
+      "sloc_bookstore",
+      "sloc_zoo_giftshop",
+      "sloc_zoo_cages",
+      "sloc_church",
+      "sloc_fire_station",
+      "sloc_police",
+      "sloc_town_hall",
+      "sloc_apartments_rooftop",
+      "sloc_apartments_rooftop2",
+      "sloc_hospital",
+      "sloc_school",
+      "sloc_hotel_rooftop",
+      "sloc_office_tower_rooftop",
+      "sloc_cathedral_rooftop"
+    ],
+    "missions": [ "MISSION_KILL_NEMESIS" ]
+  },
+  {
+    "type": "scenario",
+    "name": "Surrounded in the Dark",
+    "description": "You were camping in the wild.  One night strange noises closed in on you and you were attacked.  You managed to get away but can feel things moving in the dark.",
+    "id": "camp_attack_start",
+    "points": -2,
+    "custom_initial_date": { "hour": 2 },
+    "start_name": "Wilderness",
+    "eoc": [ "scenario_surrounded_zombie_heavy" ],
+    "allowed_locs": [
+      "sloc_campsite",
+      "sloc_cabin_lake",
+      "sloc_cabin",
+      "sloc_field",
+      "sloc_forest",
+      "sloc_campground",
+      "sloc_hermit_shack",
+      "sloc_river"
+    ],
+    "flags": [ "LONE_START", "HELI_CRASH" ]
+  },
+  {
+    "type": "scenario",
+    "id": "lone_survivor_start",
+    "name": "Lone Survivor",
+    "points": 0,
+    "description": "You are the only one that made it.  ",
```

Comment:
```suggestion
    "description": "You are the only one that made it.",
```

Reply:
Added all recommended changes to updates Scenarios.json